config/default-spinnaker-local.yml

```diff
@@ -127,9 +127,9 @@ providers:
       name: my-docker-registry
       address: ${SPINNAKER_DOCKER_REGISTRY:https://index.docker.io/}
       repository: ${SPINNAKER_DOCKER_REPOSITORY:library/nginx}
-      username: ${SPINNAKER_DOCKER_USERNAME}
+      username: ${SPINNAKER_DOCKER_USERNAME:}
       # A path to a plain text file containing the user's password
-      passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE}
+      passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE:}
```

Comment:
My interpretation was that the `:` was inserting an empty string , which Spring wasn't overriding correctly. If you can verify that this change still allows registries to authenticate when presenting `password` instead of overriding `passwordFile` in `clouddriver-local.yml` with this change in place this should be good.